modules/pages/client/components/Media.component.js

```diff
@@ -43,7 +46,10 @@ export default function Media() {
             <h4 className="media-list-divider text-muted">2017</h4>
             <dl className="dl-horizontal media-list">
               <dt>
-                <span className="text-muted">{t('October 13th')}</span>
+                <span className="text-muted">
+                  {/* October 13th */}
+                  {mediaDate(2017, 9, 13)}
```

Comment:
JS has month starting from 0 so that's why it can look odd, and that's why I added the comment for clarity.